### ISSUE TITLE
video-downloader 0.10.10: Exception: unsupported version of Python.

### ISSUE THREAD

**Issue by rjakubo** [Unrud/video-downloader#175]
Hi,

currently i use the latest version 0.10.10 of video-downloader on ubuntu 18.04.
When i try to perform a download, the following exception occurs:

![screenshot1](https://user-images.githubusercontent.com/117310959/199582276-bdf0669a-df4e-4355-9a6e-69533100bb11.png)

I perform the download also directly with the latest standalone x64 binary of yt-dlp from release 2022.10.04. 
It works fine with python 3.6.9. Therefore i don't  understand why the exception occurs. As i see, this problem 
was already reported #146. Is the check for the python version activated again?

Best regards,
Robert

**Comment by Unrud** [comment]
The problem is caused by AppArmor. It's fixed in Ubuntu 20.04. The error message is a red herring. Duplicate oft #173.

**Comment by rjakubo** [comment]
Thank's for your feedback. But I don't get the issue with AppArmor on Ubuntu 18.04. The download, e.g. of some youtube video, works without any problems when I use for that the latest standalone x64 binary of yt-dlp from release 2022.10.04.

**Comment by Unrud** [comment]
> The download, e.g. of some youtube video, works without any problems when I use for that the latest standalone x64 binary of yt-dlp from release 2022.10.04.

Yes, the issue only affects the Snap version. The Flatpak version works too.

**Comment by rjakubo** [comment]
ok
